docs-new/docs/graphql/core/api-reference/schema-metadata-api/index.mdx

```diff
@@ -0,0 +1,243 @@
+|---------------------------------------------------------------------------------------------------------------------------------------------------------------|---------------------------------------------------------------------------------------------------------------------------------------------------------|-----------|------------------------------------------------------------------|
+| **bulk**                                                                                                                                                      | [Query](#api-query) array                                                                                                                               | 1         | Execute multiple operations in a single query                    |
+| [run_sql](/graphql/core/api-reference/schema-metadata-api/run-sql.mdx#schema-metadata-run-sql)                                                                | [run_sql_args](/graphql/core/api-reference/schema-metadata-api/run-sql.mdx#schema-metadata-run-sql-args)                                                                | 1         | Run SQL directly on Postgres                                     |
+| [track_table](/graphql/core/api-reference/schema-metadata-api/table-view.mdx#schema-metadata-track-table)                                                     | [TableName](/graphql/core/api-reference/syntax-defs.mdx#tablename)                                                                                      | 1         | Add a table/view                                                 |
+| [track_table](/graphql/core/api-reference/schema-metadata-api/table-view.mdx#schema-metadata-track-table-v2)                                                  | [track_table_args](/graphql/core/api-reference/schema-metadata-api/table-view.mdx#schema-metadata-track-table-v2-syntax)                                                | 2         | Add a table/view with configuration                              |
+| [set_table_customization](/graphql/core/api-reference/schema-metadata-api/table-view.mdx#schema-metadata-set-table-customization)                             | [set_table_customization_args](/graphql/core/api-reference/schema-metadata-api/table-view.mdx#schema-metadata-set-table-customization-syntax)                           | 1         | Set table customization of an already tracked table              |
+| [set_table_custom_fields (deprecated)](/graphql/core/api-reference/schema-metadata-api/table-view.mdx#schema-metadata-set-table-custom-fields-deprecated)     | [set_table_custom_fields_args](/graphql/core/api-reference/schema-metadata-api/table-view.mdx#schema-metadata-set-table-custom-fields-deprecated-syntax)                | 2         | Set custom fields of an already tracked table (deprecated)       |
```

Comment:
hash param mismatch for `set_table_custom_fields (deprecated)` and `set_table_custom_fields_args`

- `set_table_custom_fields (deprecated)` => `#schema-metadata-set-table-custom-fields`
- `set_table_custom_fields_args` => `#schema-metadata-set-table-custom-fields-syntax`

Reply:
Done